.github/workflows/preview-env-fork.yml

```diff
@@ -0,0 +1,104 @@
+name: preview-env-fork-setup-update
+
+on:
+  workflow_run:
+    workflows: ["check-label-preview-env"]
+    types: [completed]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download PR number artifact
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: ${{ github.event.workflow_run.workflow_id }}
+          name: pr.txt
+      - name: Save PR number
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        id: pr
+        run: |
+          cat pr.txt
+          echo "::set-output name=id::$(<pr.txt)"
+      - name: Check label
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: actions/github-script@v4
+        id: label-checker
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const query = `query($owner:String!, $name:String!, $pr:Int!) {
```

Comment:
Maybe we should call a javascript file here called is-label.js? WDYT?

Reply:
you mean instead of the variable name query replace with is-label?

Reply:
Sorry, was maybe not clear enough. I thought maybe put it in an external script like a bash script. Maybe not.

Reply:
dont need to do this, but i'm 0/5 maybe can do in a follow up if is really needed

Reply:
Agreed. Not really blocking.